acquamanCommon.pri

```diff
@@ -468,7 +468,10 @@ HEADERS += \
 	source/beamline/AMSlitCenter.h \
 	source/ui/beamline/AMSlitView.h  \
 	source/beamline/AMSlits.h \
-	source/ui/beamline/AMSlitsView.h
+	source/ui/beamline/AMSlitsView.h \
+	source/beamline/AMTemperatureMonitor.h \
+    $$PWD/source/beamline/AMTemperatureMonitorGroup.h \
+    $$PWD/source/beamline/AMTemperatureMonitorGroupStatus.h
```

Comment:
Yup, I can take it out. It's added there automatically for classes that I create on my laptop, it seems.